tutornotes/templates/notes/build/notes/Dockerfile

```diff
@@ -19,8 +19,8 @@ ENV PATH /app/venv/bin:${PATH}
 # https://pypi.org/project/setuptools/
 # https://pypi.org/project/pip/
 # https://pypi.org/project/wheel/
-RUN pip install setuptools==62.1.0 pip==22.0.4 wheel==0.37.1
-RUN pip3 install -r requirements/base.txt
+RUN pip install setuptools==62.1.0 pip==22.2.1 wheel==0.37.1
```

Comment:
Can you also upgrade setuptools? https://pypi.org/project/setuptools/

Reply:
@regisb done